gems/smithy/lib/smithy/plan.rb

```diff
@@ -14,8 +14,18 @@ def initialize(model, type, options = {})
 
       Welds.load!(self)
       Polishes.load!(self)
-      @welds = Welds.for(model)
-      @polishes = Polishes.for(model)
+      @welds =
+        if options[:skip_welds]
```

Comment:
Can you add --welds [only, this] and --no-welds to CLI?

Reply:
add welds/polishes cli boolean options (which allows you to use either `--no-welds` or `--skip-welds`)